libs/openFrameworks/communication/ofSerial.cpp

```diff
@@ -446,7 +446,7 @@ bool ofSerial::setup(string portName, int baud){
 		cfgSize = sizeof(cfg);
 		GetCommConfig(hComm, &cfg, &cfgSize);
 		int bps = baud;
-		swprintf(buf, L"baud=%d parity=N data=8 stop=1", bps);
+		swprintf(buf, 80, L"baud=%d parity=N data=8 stop=1", bps);
 
```

Comment:
this is the only thing needed for the fix right?

Reply:
For the specific fix yes, but the system complained about string - fs::path conversion here, so I had to change other things to be able to find this error. if you prefer I can split this PR in two and merge this specific one

Reply:
ah cool! - no that's all good - thanks for the fix @dimitre !